server/paymentProviders/transferwise/index.ts

```diff
@@ -788,25 +789,41 @@ const oauth = {
   },
 };
 
-async function setUpWebhook(): Promise<Webhook> {
-  const url = `${config.host.api}/webhooks/transferwise`;
+const APPLICATION_TRIGGERS = ['transfers#state-change', 'transfers#refund'] as const;
+
+async function createWebhooksForHost(host = config.host.api): Promise<Webhook> {
+  const url = `${host}/webhooks/transferwise`;
   const existingWebhooks = await transferwise.listApplicationWebhooks();
 
-  if (existingWebhooks?.find(w => w.trigger_on === 'transfers#state-change' && w.delivery.url === url)) {
-    logger.info(`TransferWise App Webhook already exists for ${url}.`);
-    return;
+  for (const trigger_on of APPLICATION_TRIGGERS) {
+    if (existingWebhooks?.find(w => w.trigger_on === trigger_on && w.delivery.url === url)) {
+      logger.info(`TransferWise App Webhook already exists for ${url}.`);
+      return;
+    }
```

Comment:
`continue` rather than `return`, in case one event has a webhook setup but not the other?

```suggestion
    if (existingWebhooks?.find(w => w.trigger_on === trigger_on && w.delivery.url === url)) {
      logger.info(`TransferWise App Webhook already exists for ${url}.`);
      continue;
    }
```